W-16404049-SchedulerUpdate-EBM (#565)
mulesoft/docs-composer: en/mulesoft/modules/ROOT/pages/ms_composer_scheduling_polling_invoking_flow.adoc

To create a flow control:

. In a new flow, from the *What Should Start This Flow?* dialog, look in the Flow Control section and click *Scheduler*. 

> . Choose one of the time intervals, from 15 minutes to 30 days. If you select an interval that is a day or more, the scheduler runs once per day interval; however, it may not run at the same time each day. 

Author: ```suggestion
. Select a time interval, from 15 minutes to 30 days. For intervals of a day or more, the scheduler might not run at the same time each day.. 
```
